.github/workflows/ci-build.yml

```
@@ -0,0 +1,31 @@
+# This workflow will do a clean install of node dependencies, build the source code and run tests across different versions of node
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+
+name: Node.js CI
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
```

@stevengill As the default timeout is 6 hours, add a shorter timeout like this (just in case).
```suggestion
    runs-on: ubuntu-latest
    timeout-minutes: 10
```